<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionSettingsDialog</class>
 <widget class="QDialog" name="SessionSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Session Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Session</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Operator password:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="opword">
            <property name="text">
             <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;not set&lt;/span&gt; (&lt;a href=&quot;#change&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;assign&lt;/span&gt;&lt;/a&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Autoreset threshold:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QDoubleSpinBox" name="autoresetThreshold">
              <property name="toolTip">
               <string>Session history size at which autoreset triggers</string>
              </property>
              <property name="specialValueText">
               <string>No autoreset</string>
              </property>
              <property name="suffix">
               <string> MB</string>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="baseResetThreshold">
              <property name="text">
               <string notr="true">+ 0 MB</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Options:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="preserveChat">
            <property name="toolTip">
             <string>Include chat in session history so new users can see it as well</string>
            </property>
            <property name="text">
             <string>Keep chat history</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="persistent">
            <property name="toolTip">
             <string>Don't terminate session when the last user leaves</string>
            </property>
            <property name="text">
             <string>Persist without users</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="nsfm">
            <property name="toolTip">
             <string>This session contains material not suitable for minors</string>
            </property>
            <property name="text">
             <string>Age restricted content</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLineEdit" name="title"/>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" rowspan="2" colspan="3">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Login</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="sessionPassword">
            <property name="text">
             <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;not set&lt;/span&gt; (&lt;a href=&quot;#change&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;assign&lt;/span&gt;&lt;/a&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Max users:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="maxUsers">
            <property name="toolTip">
             <string>How many users to let in. Does not affect users already logged in</string>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>254</number>
            </property>
            <property name="value">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="denyJoins">
            <property name="text">
             <string>Block new logins</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="authOnly">
            <property name="text">
             <string>No guest logins</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Permissions</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Cut, Paste &amp; Fill:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="permPutImage"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Selection Move &amp; Resize:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="permRegionMove"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>New Annotations:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="permCreateAnnotation"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>General Layer Editing:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="permEditLayers"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Own Layer Editing:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="permOwnLayers"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Canvas Resizing:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="permResize"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Canvas Background:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="permBackground"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Laser Pointer</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="permLaser"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Undo &amp; Redo:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="permUndo"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Kick &amp; Ban:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="deputies">
         <item>
          <property name="text">
           <string>Operators</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Trusted</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Banned users</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableView" name="banlistView">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="removeBan">
           <property name="text">
            <string>Remove selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Announcements</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableView" name="announcementTableView">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addPrivateAnnouncement">
           <property name="text">
            <string>Private listing</string>
           </property>
           <property name="icon">
            <iconset theme="list-add">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addAnnouncement">
           <property name="text">
            <string>Public listing</string>
           </property>
           <property name="icon">
            <iconset theme="list-add">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeAnnouncement">
           <property name="text">
            <string>Remove selected</string>
           </property>
           <property name="icon">
            <iconset theme="list-remove">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SessionSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SessionSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
